test/attributes.test.ts

```diff
@@ -22,7 +30,7 @@ let client1Address: string;
 let client2Address: string;
 
 beforeAll(async () => {
-    [client1, client2] = await launcher.launch(2);
+    [client1, client2] = await launcher.launch(3);
```

Comment:
you launch 3 but only use 3, any reason for that?

Reply:
nop just a missing undo from an earlier implementation